CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+## Reporting Issues
+
+If you have found what you think is a bug, please [file an issue](https://github.com/pmndrs/jotai/issues/new).
+
+For usage questions, prefer [starting a discussion](https://github.com/pmndrs/jotai/discussions/new).
+
+## Suggesting new features
+
+If you are here to suggest a feature, first create an issue if it does not already exist. From there, we will discuss use-cases for the feature and then finally discuss how it could be implemented.
+
+## Development
+
+If you have been assigned to fix an issue or develop a new feature, please follow these steps to get started:
+
+- Fork this repository.
+- Install dependencies by running `$ yarn`.
+- Implement your changes and run test files or create new test files.
+- If you want to test it in a React project you can either use `$ yarn link` or `yalc` package.
+- Git stage your required changes and commit (see below commit guidelines).
+- Submit PR for review.
+
+### Type
+
+Must be one of the following:
+
+- **feat**: A new feature
+- **fix**: A bug fix
+- **docs**: Documentation only changes
+- **refactor**: A code change that neither fixes a bug nor adds a feature
+- **test**: Adding missing or correcting existing tests
+- **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
+  generation
+
+## Pull requests
+
+Maintainers merge pull requests by squashing all commits and editing the commit message if necessary using the GitHub user interface.
+
+Use an appropriate commit type. Be especially careful with breaking changes.
```

Comment:
Is this about pull request title, or commit message? What would one do if there's a breaking change?

Reply:
dear @dai-shi, I have removed this part.